client/schema/v1/index.js

```diff
@@ -892,14 +892,16 @@ each other.`,
           },
           steps: {
             title: 'Steps',
-            hint: 'A step can be a single procedure or a combination of procedures to achieve an outcome. You will be able to reorder your steps at any time before you send your application to the Home Office.',
+            hint: 'A step can be a single procedure or a combination of procedures to achieve an outcome. You will be able to reorder your steps at any time before you send your application to the Home Office, but they should be broadly chronological, with the final step being a method of killing or the last regulated procedure.',
             footer: 'Once you’ve created a list of steps, you need to add information about adverse effects, controls and limitations, and humane endpoints to each one.​',
             fields: [
               {
                 name: 'title',
                 type: 'texteditor',
                 label: 'Describe the procedures that will be carried out during this step.',
-                hint: 'To ensure that an adequate harm benefit assessment can be carried out, please provide a clear and explicit explanation of each step.'
+                hint: `Explain where one or more steps are repeated in one experiment, list any alternative techniques within a step (e.g. dosing routes), and include all procedures performed under terminal anaesthesia.
+
+                When describing the technical aspects of a step, be broad enough to be flexible when the variation does not impact on animal welfare (e.g. use "antibiotic" instead of "penicillin").  Finally, avoid specifying  volumes and frequencies when they do not impact on animal welfare.`
```

Comment:
So this has not worked, but it's not worked in a very curious way.

Will need to take a closer look.